src/css/_utilities.scss

```diff
@@ -115,3 +115,10 @@
   @extend %fill-parent;
   text-align: center;
 }
+
+// don't wrap whitespace for the time control and make the text container 0 width
+// fixes a bug with 'width: auto' in IE 8/9
+.vjs-ie-auto-width-fix {
```

Comment:
Sorry, I meant, can we rely on the vjs-no-flex class rather than introducing another one? i.e., move the properties from this selector into that?